plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/KubernetesClusterManagerImpl.java

```diff
@@ -832,6 +892,9 @@ private void validateKubernetesClusterScaleParameters(ScaleKubernetesClusterCmd
 
         Account caller = CallContext.current().getCallingAccount();
         accountManager.checkAccess(caller, SecurityChecker.AccessType.OperateEntry, false, kubernetesCluster);
+        if (!kubernetesCluster.getManaged()) {
+            throw new InvalidParameterValueException(String.format("Kubernetes cluster %s is not managed by Apache CloudStack", kubernetesCluster.getName()));
+        }
```

Comment:
the exact same bit of code is used to throw the exact same exception later on. please extract to a method or chencge the messages.